kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/main/java/org/kie/server/integrationtests/shared/KieServerSynchronization.java

```diff
@@ -115,6 +121,71 @@ public boolean conditionPassed() {
         });
     }
 
+    public static void waitForContainerWithReleaseId(final KieServicesClient client, final ReleaseId releaseId) throws Exception {
+        waitForCondition(new WaitingCondition() {
+            @Override
+            public boolean conditionPassed() {
+                ServiceResponse<KieContainerResourceList> containersList = client.listContainers();
```

Comment:
You may use here listContainers(KieContainerResourceFilter containerFilter) to retrieve just containers with specific release id.